index.html

```diff
@@ -0,0 +1,195 @@
+    src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"
+  ></script>
+  <link rel="stylesheet" href="page.css">
+  <title>African Youth in Tech Forum Home Page | Stanley</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="top-nav">
+      <a href="#">
+        <ion-icon class="top-nav-icon" name="logo-facebook"></ion-icon>
+      </a>
+      <a href="#"><ion-icon class="top-nav-icon" name="logo-twitter"></ion-icon>
+      </a>
+      <a href="#">
+        <ion-icon class="top-nav-icon" name="logo-whatsapp"></ion-icon>
+      </a>
+      <a href="#">
+        <ion-icon class="top-nav-icon" name="logo-instagram"></ion-icon>
+      </a>
+      <a href="#">English</a>
+      <a href="#">My Page</a>
+      <a href="#">Log-Out</a>
+    </nav>
+    <nav class="bottom-nav">
+      <a href="index.html">
+        <img src="images/logo-img.png" alt="youth in tech logo" class="web-logo" width="50" height="50">
+      </a>
+      <ul class="navbar-desktop">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#program">Program</a></li>
+        <li><a href="#join-us">Join</a></li>
+        <li><a href="#partners">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li class="tech-campaign">
+          <a href="#">AYT Forum</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <nav class="mobile-menu">
+    <div class="nav-menu">
+      <a href="index.html" class="mobile-menu-logo">
+        <img class="menu-logo" src="images/logo-img.png" alt="youth in tech logo" width="80" height="80"
+        >
+      </a>
+      <img src="icons/Icon-cancel.svg"
+      alt="close button"
+      class="close-button"
+      width="30"
+      height="30">
+    </div>
+    <ul class="menu-ul">
+      <li id="nav-item"><a href="about.html">About</a></li>
+      <li id="nav-item"><a href="#program">Program</a></li>
+      <li id="nav-item"><a href="#join-us">Join</a></li>
+      <li id="nav-item"><a href="#partners">Sponsor</a></li>
+      <li id="nav-item"><a href="#">News</a></li>
+      <li id="nav-item" class="tech-campaign"><a href="#" >AYT Forum</a></li>
+    </ul>
+  </nav>
+  <main>
+    <section class="mobile-container-one">
+      <div class="navbar-mobile">
+        <img src="icons/Icon-Menu.png" alt="hambuger menu">
+      </div>
+      <div class="mobile-one-content">
+        <p class="greetings-mobile">  "Hello! African Youths"</p>
+        <h1>2023 African Youth in Tech Forum</h1>
+        <p class="details">
+          A unique opportunity for the African youths and other young people across the world to learn from the best minds in the world of technology. The event is a tech forum that will be taking place in February, 2023 for 3 days from 3rd Friday to 5th Sunday.
+        </p>
+        <h2>03.01.2023(FRI)~5(SUN)</h2>
+        <p class="venue">@ Kenyatta International Convention Centre, Nairobi, Kenya</p>
+      </div>
+    </section>
+    <section id="program" class="main-program-section"> 
```

Comment:
- [ ] @chepkok3 as previously requested kindly ensure your design matches the design given, you can achieve it by setting the width of the image title and text to a set limit so that the get aligned as in the design given.